akd_core/src/lib.rs

```diff
@@ -182,12 +182,21 @@ extern crate alloc;
 #[cfg(all(feature = "protobuf", not(feature = "nostd")))]
 pub mod proto;
 
-pub mod crypto;
 pub mod ecvrf;
 pub mod hash;
 pub mod utils;
 pub mod verify;
 
+pub mod configuration;
+pub use configuration::Configuration;
+
+// Note(new_config): Update this when adding a new configuration
+
+#[cfg(feature = "experimental")]
```

Comment:
rather than importing it as a macro or something, do we just want to do

```rust
#[cfg(feature = "experimental")]
pub use configuration::experimental::ExperimentalConfiguration as TheConfiguration;
#[cfg(feature = "whatsapp_v1")]
pub use configuration::whatsapp_v1::WhatsAppV1Configuration as TheConfiguration;
```

Then we can just refer to `akd_core::TheConfiguration` everywhere, which will map to the right type

Reply:
See my above comment on features needing to be additive (so that enabling any combination of features should always compile / not break things)